VerloopSDK/VLConfig.swift

```diff
@@ -94,25 +184,23 @@ public typealias LiveChatUrlClickListener = (_ url : String?)  -> Void
         urlRedirection = flag
     }
 
-    
     @objc public func putCustomField(key: String, value: String, scope: SCOPE) {
-        
-        switch scope {
-        case .USER:
-            customFields.append(CustomField(key: key, value: value, scope: "user"))
-        case .ROOM:
-            customFields.append(CustomField(key: key, value: value, scope: "room"))
+        if !key.hasEmptyValue(),!value.hasEmptyValue() {
+            updatedConfigParams.append(.customFields)
+            switch scope {
+                case .USER:
+                    customFields.append(CustomField(key: key, value: value, scope: "user"))
+                case .ROOM:
+                    customFields.append(CustomField(key: key, value: value, scope: "room"))
```

Comment:
If the scope is not provided, it is room scope

Reply:
Its an enum with two possible values. I'd added a default on the switch case.